### PR TITLE
Remove update in background button

### PR DESCRIPTION
**Pull request by mehtaamish89**
For https://github.com/Medology/raven-scanner-app/issues/53 & https://github.com/Medology/raven-scanner-app/issues/157

- Removed the piece of code which allows the plugin to update the app in the background.
- Fixed some grammatical mistakes.

### CODE REVIEW

File: res/values-en/appupdate_strings.xml [hunk 1]
```diff
@@ -13,6 +13,6 @@
     <string name="download_complete_neu_btn">Install manually</string>
 
     <string name="update_error_title">Error</string>
-    <string name="update_error_message">The current network is unavailable , please check your network settings.</string>
+    <string name="update_error_message">The current network is unavailable, please check your network settings.</string>
```
Comment (goodevilgenius) [inline res/values-en/appupdate_strings.xml]: This wording is still a little awkward. "The current network is unavailable" might be better written as "Network connection is unavailable" or "Internet connection is not available."

File: src/android/MsgBox.java [hunk 2]
```diff
@@ -18,109 +18,108 @@
...
+  private ProgressBar downloadDialogProgress;
+  private Dialog errorDialog;
+
+  public MsgBox(Context mContext) {
+    this.mContext = mContext;
+    this.msgHelper = new MsgHelper(mContext.getPackageName(), mContext.getResources());
+  }
+
+  /**
+   * 显示软件更新对话框
```
Comment (goodevilgenius) [inline src/android/MsgBox.java]: ‽ 😕

Please translate comments to English.

- [x] Done
- [x] Approved

File: src/android/MsgBox.java [hunk 3]
```diff
@@ -18,109 +18,108 @@
...
-            builder.setPositiveButton(msgHelper.getString(MsgHelper.UPDATE_UPDATE_BTN), onClickListener);
-            noticeDialog = builder.create();
-        }
-
-        if (!noticeDialog.isShowing()) noticeDialog.show();
-
-        noticeDialog.setCanceledOnTouchOutside(false);// 设置点击屏幕Dialog不消失
-        return noticeDialog;
+    if (!noticeDialog.isShowing())
+      noticeDialog.show();
```
Comment (goodevilgenius) [inline src/android/MsgBox.java]: I'd feel a lot more comfortable with `{}` here.

- [x] Done
- [x] Approved

File: src/android/MsgBox.java [hunk 4]
```diff
@@ -18,109 +18,108 @@
...
+
+    noticeDialog.setCanceledOnTouchOutside(false);// 设置点击屏幕Dialog不消失
+    return noticeDialog;
+  }
+
+  /**
+   * 显示软件下载对话框
+   */
+  public Map<String, Object> showDownloadDialog(OnClickListener onClickListenerNeg, OnClickListener onClickListenerPos,
+      OnClickListener onClickListenerNeu, boolean showDialog) {
```
Comment (goodevilgenius) [inline src/android/MsgBox.java]: Maybe split this up a bit more. This is really difficult to read. The original was easier.

- [x] Done
- [x] Approved

File: src/android/MsgBox.java [hunk 5]
```diff
@@ -18,109 +18,108 @@
...
+      builder.setTitle(msgHelper.getString(MsgHelper.UPDATING));
+      // 给下载对话框增加进度条
+      final LayoutInflater inflater = LayoutInflater.from(mContext);
+      View v = inflater.inflate(msgHelper.getLayout(MsgHelper.APPUPDATE_PROGRESS), null);
+
+      /* 更新进度条 */
+      downloadDialogProgress = (ProgressBar) v.findViewById(msgHelper.getId(MsgHelper.UPDATE_PROGRESS));
+      builder.setView(v);
+      // 取消更新
+      // builder.setNegativeButton(msgHelper.getString("update_cancel"),
```
Comment (goodevilgenius) [inline src/android/MsgBox.java]: Delete unused code.

- [x] Done
- [x] Approved

**Review by goodevilgenius — CHANGES_REQUESTED** [review]

**Review by Chekote — APPROVED** [review]

File: res/values-en/appupdate_strings.xml (hunk 1 above)
Comment (mehtaamish89) [reply res/values-en/appupdate_strings.xml]: 👍 Done

File: src/android/MsgBox.java (hunk 3 above)
Comment (mehtaamish89) [reply src/android/MsgBox.java]: Rather than having brackets I have added the if the statement to the same line as the condition.

**Review by goodevilgenius — APPROVED** [review]

**Pull request merged**